libtransact/src/protocol/batch.rs

```diff
@@ -118,7 +118,7 @@ impl Batch {
     }
 }
 
-#[derive(Debug, PartialEq)]
+#[derive(Clone, Debug, PartialEq)]
```

Comment:
Perhaps, instead of adding clone here, the error that has a batch pair should just have the batch header signature, since that is all it uses in its Display implementation.

Reply:
Good catch; fixed